main/modes/games/cGrove/Garden/cg_Grove.c

```diff
@@ -57,26 +60,35 @@ static const char* speechBubbleSprites[] = {
     "cg_text3.wsg",
 };
 
+static const char* eggsIntactSprites[] = {
+    "chowa_egg1.wsg",
+    "chowa_egg2.wsg",
+};
+
+static const char* eggsCrackedSprites[] = {
+    "chowa_egg_hatch1.wsg",
+    "chowa_egg_hatch2.wsg",
+};
+
 static const char* itemSprites[] = {
     "agi_book.wsg",   "cha_book.wsg", "spd_book.wsg",     "sta_book.wsg", "str_book.wsg", "cg_ball.wsg",
     "cg_crayons.wsg", "cg_knife.wsg", "cg_toy_sword.wsg", "cake.wsg",     "souffle.wsg",  "DonutRing.wsg",
 };
 
-static const char shopMenuTitle[] = "Ring Shop";
-
-static const char* shopMenuItems[] = {
-    "Agility Stat Book",
-    "Charisma Stat Book",
-    "Strength Stat Book",
-    "Stamina Stat Book",
-    "Speed Stat Book",
-    "Souffle",
-    "Ball",
-    "Crayons",
-    "Toy Sword",
-    "Back",
+static const char shopMenuTitle[] = "Grove Menu";
+
+static const char* menuLabels[] = {
+    "Shop for items", "View inventory", "View Chowa", "Release Chowa", "Tutorial", "Exit Menu", "Exit Grove",
 };
 
+static const char* nvsBlobKeys[] = {"cgItems", "cgInventory", "cgEggs"};
+
+static const char nvsTutorialKey[] = "cgGroveTut";
+
+static const char* cgNVSKeys[] = {"cgPlayerName", "cgChowaData", "cgSettings", "cgGuestData"};
```

Comment:
This is duplicated in `mode_cGrove.c`

Reply:
Yeah, I couldn't figure out how to get it to build otherwise. It kept saying it wasn't used if it was in the typedef file, but it had issues with redefining even if I only put it into cg_Grove.c/.h and tried to reference it. If you have a solution, I'm all ears, I just wanted it  to actually build and work

Reply:
There's two solutions. One is to use a getter, just a function that returns `cgNVSKeys`. The other is to declare `cgNVSKeys` as `extern` in the header, which tells the linker that the variable exists in this object file so that other object files can reference it. `extern` can get really messy for scoping variables, but for `const` stuff it's not so bad.